src/test/java/teammates/test/cases/browsertests/FeedbackMsqQuestionUiTest.java

```diff
@@ -509,6 +512,110 @@ public void testDeleteQuestionAction() {
         assertNull(BackDoor.getFeedbackQuestion(courseId, feedbackSessionName, 1));
     }
 
+    /**
+     * Tests that MSQ options (new and existing) can be reordered using drag and drop mechanism.
```

Comment:
nit: `Tests` --> `Verifies` (Reason: at the first reading `Tests` can be mistaken for plural of `Test`)